apps/public/src/app/(general)/samenvatting/_utils/getSummaryData.ts

```diff
@@ -1,55 +1,90 @@
-import type { GetMeldingByMeldingIdAnswersResponse, MeldingOutput } from '@meldingen/api-client'
+import {
```

Comment:
This name of this file is a little off. It implies that it contains a function called `getSummaryData`, but it actually contains a set of functions. We could break the functions into separate files (which is easier to test) or maybe a `utils.tsx` in `/samenvatting`